src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      return a / b;
+    },
 
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
```

Comment:
[NAMING] - The `operate` method correctly uses the provided callback and number to set the `result`, but ensure that the callback functions are updated to only take one parameter and operate on the `result` property directly.